enhancements/declarative-index-config.md

```diff
@@ -865,34 +865,22 @@ $ tree
 ├── etcd.json
 ├── amqstreams.json
 ```
-#### Editing a package inside an index
+#### Creating new indexes using the config files/Editing a package inside an existing index
 
-A new sub-command `update` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input along with an existing index image tag, and will replace the edited config files in the index container. 
+A new sub-command `build` will be introduced under `opm index`, which will take a folder containing json representations of packages as it's input, and will build a container image with the folder and the `opm` binary in it. 
```

Comment:
Will this command will _only_ build images in-process or will it also be able to exec Docker/podman/etc?

Reply:
I was thinking about the second option, i.e exec Docker/podman etc. I can make that more explicit in the doc.

Reply:
hmm, IMO the increased support burden of execing external tools isn't worth it.

Reply:
Per [this comment](https://github.com/operator-framework/enhancements/pull/68#discussion_r590646373), I do see the value of carrying support for execing external tools, and I'm guessing that is originally why we decided to include it in the first place.